cmd/quickstart.go

```diff
@@ -116,6 +120,11 @@ func quickstartExec(ctx context.Context, flags QuickstartFlags) error {
 		quickstartObj.Defaults.TargetType = &flags.TargetType
 	}
 
+	if flags.From != "" {
+		quickstartObj.Defaults.Blueprint = &flags.From
+		quickstartObj.IsUsingBlueprint = true
```

Comment:
Blueprint isn't super descriptive?

Reply:
This would be cleaner if we didn't have this special case, instead if we're using the from argument -> set the schema URL to our schema store URL cache thing

Reply:
No other code path is configured to parse a url - it's all file based, hence this special casing